### PR TITLE
feat: Implemented PlaybackStream

### PR DESCRIPTION
**Pull request by vishal-sehgal**
Fixes #17 
Ported Java Code.

**Changes**: Implemented PlaybackStream.

**Checklist**:
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications are done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

### CODE REVIEW

**Comment by vishal-sehgal** [comment]
@iamareebjamal @jddeep  Please review this PR.

**Comment by vishal-sehgal** [comment]
@iamareebjamal

File: app/src/main/java/io/neurolab/PlaybackStream.java [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+                double[] currentSample = new double[4];
+                for (int c = 0; c < this.numberOfChannels; c++) {
+                    currentSample[c] = 12.5d * Math.sin(((double) (i + c * 16) / 128d) * Math.PI * 6d);
+                    currentSample[c] += 12.5d * Math.sin(((double) (i + c * 16) / 128d) * Math.PI * 25d);
+                }
+                currentSamples.add(currentSample.clone());
+                this.data.add(currentSamples);
+            }
+        } else {
+            try {
```
Comment (iamareebjamal) [inline app/src/main/java/io/neurolab/PlaybackStream.java]: Use try with resources

Comment (vishal-sehgal) [reply app/src/main/java/io/neurolab/PlaybackStream.java]: Can you please elaborate more?

Comment (iamareebjamal) [reply app/src/main/java/io/neurolab/PlaybackStream.java]: Read about try with resources

Comment (vishal-sehgal) [reply app/src/main/java/io/neurolab/PlaybackStream.java]: Okay thanks, I will check it out and implement it 👍

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**